docs/machine-learning/toc.yml

```
@@ -97,4 +97,4 @@
     - name: Use the automated ML API
       href: how-to-guides/how-to-use-the-automl-api.md
     - name: API reference
-      href: https://docs.microsoft.com/en-us/dotnet/api/index?view=automl-dotnet
\ No newline at end of file
+      href: https://docs.microsoft.com/en-us/dotnet/api/index?view=automl-dotnet
```

Caught the eye, not related to this PR, though:
```suggestion
      href: https://docs.microsoft.com/dotnet/api/index?view=automl-dotnet
```

---

Good catch @pkulikov . Thanks!